src/main/java/com/jalasoft/jfc/model/utility/PathJfc.java

```diff
@@ -57,39 +57,39 @@ public PathJfc() throws IOException {
     }
 
     /**
-     * This gets the Image Magick's path.
+     * Gets the Image Magick's path.
```

Comment:
you could improve "Gets the Image Magick's path" to **Gets Image Magick's path.**